src/Cli/dotnet/commands/dotnet-complete/Suggest.cs

```diff
@@ -13,37 +13,55 @@ internal static class Suggest
     {
         public static IEnumerable<string> TargetFrameworksFromProjectFile()
         {
-            var msBuildProject = GetMSBuildProject();
-
-            if (msBuildProject == null)
+            try
             {
-                yield break;
+                return GetMSBuildProject()?.GetTargetFrameworks().Select(tf => tf.GetShortFolderName()) ?? Empty<string>();
```

Comment:
Thanks for the quick turnaround. Although I don't know how that worked. Why the change is in dotnet-complete? Isn't that only for auto competition?

Reply:
The problem was with generating argument suggestions when the project file isn't valid, which occurs regardless of which command is called when the parser is instantiated. Here's an example of how these methods are used: https://github.com/dotnet/sdk/blob/master/src/Cli/dotnet/CommonOptions.cs#L39